src/axom/sidre/core/Group.cpp

```diff
@@ -3212,15 +3358,12 @@ bool Group::rename(const std::string& new_name)
  *
  *************************************************************************
  */
-int Group::getValidAllocatorID(int allocID)
+int Group::getValidAxomAllocatorID(int allocID)
```

Comment:
Note: This is an API change -- but it is a private function

Reply:
The method has been removed.